Add information on withoutMiddleware and rate limit throttling (#197)
wintercms/docs: services/router.md

});
```

If you want to assign a group of middleware to just one route you can it like this

```php
Route::middleware(['Path\To\Your\Middleware'])->group(function() {
    Route::get('info', 'Acme\News@info');
});
```

And finally, if you want to remove middleware from a route you can it like this

```php

> Route::withoutMiddleware("Path\To\Removed\Middleware")

Reviewer: ```suggestion
Route::withoutMiddleware(\Path\To\Removed\Middleware::class)
```

Author: Nice. I'll try making that change in my code.

Reviewer: It's a lot nicer for IDEs so I'm trying to gradually shift all the core code and examples in that direction.

    ->get('info', 'Acme\News@info');
```

You can of course add more than one middleware in a group, just one is used in the above examples for convenience.

## Throttle middleware

> When using Winter's API middleware, you can remove the default 60 request per minute rate throttle applied to Winter's
> API middleware and replace it with your own. For example

Reviewer: ```suggestion
When using Winter's `api` middleware, you can remove the default rate limiting of 60 requests per minute and replace it with your own by using the below example:
```
